lib/config/file.ts

```diff
@@ -14,6 +14,11 @@ export function getConfig(env: NodeJS.ProcessEnv): RenovateConfig {
     // eslint-disable-next-line global-require,import/no-dynamic-require
     config = require(configFile);
   } catch (err) {
+    // istanbul ignore if
+    if (err instanceof SyntaxError) {
+      logger.fatal(`Could not parse config file \n ${err.stack}`);
+      return null;
```

Comment:
Let's quit out (abort) instead of continuing. Otherwise a single typo could result in a complete change of config for 10s/100s/1000s of repos.

Reply:
I found out
1. Even if the config is wrong, with a valid token renovate continues to operate.That doesn't the fix for the problem.

2. I added the return to prevent the continuation


Do you want me to have process.exit(1) there ?

Reply:
Yes, process.exit

Reply:
Made the changes accordingly.Its the best way to solve this error
Also, the error message is more clear now.